src/workers/metrics_capture.rs

```diff
@@ -10,14 +10,6 @@ use serde::{Deserialize, Serialize};
 
 lazy_static::lazy_static! {
     static ref TXS_IN_STORE: GenericGauge<prometheus::core::AtomicI64> = register_int_gauge!(opts!("literpc_txs_in_store", "Transactions in store")).unwrap();
-    static ref TOKIO_TASKS: GenericGauge<prometheus::core::AtomicI64> = register_int_gauge!(opts!("literpc_tokio_tasks", "Tokio tasks in lite rpc")).unwrap();
```

Comment:
Could you disable this code depending on the rust flag. If tokio_unstable is active I want to enable this code.

Reply:
I want to reactivate the code if necessary.

Reply:
lemme see